resin/models/application.py

```diff
@@ -56,10 +57,13 @@ def get(self, name):
             'eq': name
         }
         try:
-            return self.base_request.request(
+            apps = self.base_request.request(
                 'application', 'GET', params=params,
                 endpoint=self.settings.get('pine_endpoint')
-            )['d'][0]
+            )['d']
+            if len(apps) > 1:
```

Comment:
https://github.com/resin-io/resin-sdk/blob/master/lib/models/application.coffee#L146-L147. It's added in JS SDK.
This can happen when you log in as an admin but not sure how the user can have this (I checked our dashboard and we do have a check for application name)